classes/suggested-tasks/class-remote-tasks.php

```diff
@@ -106,6 +106,6 @@ protected function get_api_endpoint() {
 				$url
 			);
 		}
-		return \apply_filters( 'progress_planner_suggested_tasks_remote_api_endpoint', $url );
```

Comment:
I think we added this filter also to allow 3rd party integration (ie hosts), but we can always add it back when that is thought through.

Reply:
Yeah, if I'm not mistaken we'll be implementing an alternative method using our site as a proxy. Still under discussion and not urgent at this stage, but we don't need the filter for now.